### PR TITLE
Java formatting fixes

### PR DESCRIPTION
**Pull request by nrgraham23**
Fixed tab indenting as well as some of the block comments
at the top of a few of the files.

White space/formatting fixes

Signed-off-by: Nathaniel Graham ngraham@lanl.gov

@hppritcha @jsquyres

:bot:milestone:v2.0.0
;bot:label:enhancement

### CODE REVIEW

File: ompi/mpi/java/java/CartComm.java [hunk 1]
```diff
@@ -52,181 +52,181 @@
...
+		return new CartComm(dup(handle));
+	}
+
+	/**
+	 * Duplicates this communicator.
+	 * <p>Java binding of {@code MPI_COMM_IDUP}.
+	 * <p>The new communicator can't be used before the operation completes.
+	 * The request object must be obtained calling {@link #getRequest}.
+	 * @return copy of this communicator
+	 * @throws MPIException Signals that an MPI exception of some sort has occurred. Signals that an MPI exception of some sort has occurred.
```
Comment (hppritcha) [inline ompi/mpi/java/java/CartComm.java]: two instances of Signals that an MPI....

**Comment by hppritcha** [comment]
good except for the double instance of the "Signals that an MPI exception" string problem.

File: ompi/mpi/java/java/CartComm.java (hunk 1 above)
Comment (nrgraham23) [inline ompi/mpi/java/java/CartComm.java]: Fixed

**Comment by hppritcha** [comment]
:+1:

**Pull request merged**